service/history/handler_test.go

```diff
@@ -23,13 +23,15 @@ package history
 import (
 	"context"
 	"errors"
-	"github.com/stretchr/testify/mock"
-	"github.com/uber/cadence/common/metrics/mocks"
-	"go.uber.org/yarpc/yarpcerrors"
 	"math/rand"
 	"sync/atomic"
 	"testing"
 
+	"github.com/stretchr/testify/mock"
+	"go.uber.org/yarpc/yarpcerrors"
+
+	"github.com/uber/cadence/common/metrics/mocks"
+
```

Comment:
inconsistent new lines in the import section. One common practice that I followed in the past is to group imports like this:
- builtin imports (e.g. fmt)
- local imports (stuff from current repo)
- external imports (vendor libraries)

There would be new line between each group.
I know we don't follow everywhere this but it can help organize imports when unsure.